hicexplorer/hicExport.py

```diff
@@ -184,7 +184,7 @@ def main(args=None):
             hic_ma.distance_counts = distance_counts
 
     else:
-        if args.inputFormat == 'cool' and args.chromosomeOrder is not None and len(args.chromosomeOrder) == 1:
+        if args.inputFormat == 'cool' and args.chromosomeOrder is not None and len(args.chromosomeOrder) > 0:
```

Comment:
We have to use `==  1` because we can only use the benefits of the cooler format to load the matrix partial if we load one position. More is so far not possible. I had the idea to load each region and to merge this together to one matrix. This failed so far because you get a different size of the matrix and the number of bins. Therefor the concept is to load one chromosome or one region and use the benefits of cooler, in the case we have to load more, we open the cooler file, load all data and reduce it to what is needed (same way as before with h5)

Reply:
Indeed, here (line 187) the condition is fine!

Reply:
Now I understand why it needs to be `==1` I was not sure when I made the chance.